### ISSUE TITLE
How to use string data in benchmark?

### ISSUE THREAD

**Issue by mocenas** [Hyperfoil/Hyperfoil#105]
I need to use string as input to my tested system - long text, lets say length of one paragraph.
How can I do this with hyperfoil? 

I'm completely fine with putting these data it into some separate data file. But how to read this file in benchmark? Only function I found, that can read files is `randomCsvRow`, it is a bit clumsy for this use case, but if there is nothing else, it'll do. How can I upload this data file to hyperfoil in clustered mode, so it will be associated with correct benchmark?

Or is there any other solution how to generate string test data?

**Comment by rvansa** [comment]
As you've noticed, there's the `body` parameter; you can post the multiline string directly in the YAML. I am sorry about the bug; right now you can work-around using
```
- httpRequest:
     ...
     body:
       text: |+
         This is a multiline string
          that will be pasted as the body,
          including newlines.
```
Reading that from file is not implemented ATM, though it's pretty easy to do; add a method (`fromFile`) to `HttpRequestStep$BodyBuilder`. Check out this place: https://github.com/Hyperfoil/Hyperfoil/blob/903498afa5a2b810529be897e3e24d45758547af/core/src/main/java/io/hyperfoil/core/generators/RandomItemStep.java#L157 to see how you read a file and register it with the benchmark so that the server keeps the file along.

**Comment by rvansa** [comment]
Implemented in 2905d858dc8a163416ef77be695f609de9f430cc
